go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/OneOfOne/xxhash v1.2.8
 	github.com/bytecodealliance/wasmtime-go v0.35.0
+	github.com/containerd/containerd v1.6.2
```

Comment:
This is going to pull down a lot of packages, isn't it ? Do we need to do this or is there some particular package / code segment that we are using ?

Reply:
Some background: to keep OPA's dependencies light, we've resorted to "manual vendoring" of third party code. Like, we've copied over some jwt lib to back the jwt built-ins. If there is only a manageable amount of code needed from a third party dependency here, we could do the same. I haven't dug into this yet, so I don't know the answer.

Reply:
Hrmm looking at this briefly, the oras packages have a [`pkg/auth/docker`](https://pkg.go.dev/oras.land/oras-go@v1.1.1/pkg/auth/docker), too. I'd expect that if we used that _instead_ of "github.com/containerd/containerd/remotes/docker", we'd be able to cut down the dependencies a bit? WDYT @carabasdaniel?

Reply:
We are using the docker resolver [here](https://github.com/aserto-dev/opa/blob/oci/download/oci_dowload.go#L304) as the source target for the oras.Copy method.  At the moment I'm not sure we have a lighter solution for this.

Reply:
So my last clue here is probably a dead-end: https://github.com/oras-project/oras/blob/main/go.mod#L6 ORAS comes with the containerd/containerd module as a dependency, too.

Reply:
@carabasdaniel sorry I had missed your comment when I posted my last one -- here's a follow-up: https://github.com/open-policy-agent/opa/pull/4558#discussion_r860611612